lib/apicache.js

```diff
@@ -108,6 +108,10 @@ function ApiCache() {
       }
 
       if (cached = memCache.get(key)) {
+        if (_.isFunction(bypassCache) && bypassCache(req, res)) {
```

Comment:
Why wait till the cache is checked before bypassing?  This looks like the same functionality as has been implemented in middlewareToggle (except boolean reversal), so I'm closing the issue...

Thanks for the suggestion though!